src/core/logical_layers/renderers/BivariateRenderer/types.ts

```diff
@@ -0,0 +1,6 @@
+export type FillColor = {
```

Comment:
I'd move it to core/types and reuse in /src/utils/bivariate/bivariateColorThemeUtils.ts#L229
also it can be named more generic - ColorRGBA